### ISSUE TITLE
[UI] Multifactor token configuration

### ISSUE THREAD

**Issue by SchrodingersGat** [inventree/InvenTree#8708]
Multifactor authentication setup is missing in the new admin center:

![Image](https://github.com/user-attachments/assets/56098663-7eaa-427c-af27-98bfc12da7ef)

**Comment by matmair** [comment]
Not sure about this, I think I worked on this a while ago. Will try to reproduce.

**Comment by SchrodingersGat** [comment]
Yeah, I thought you had also... Maybe there's a config option I'm missing?

**Comment by matmair** [comment]
It seems this was not really fully implemented, I will look into it as part of https://github.com/inventree/InvenTree/pull/6293
